libmscore/stafftype.cpp

```diff
@@ -241,6 +243,7 @@ void StaffType::write(Xml& xml) const
             xml.tag("stemsDown",        _stemsDown);
             xml.tag("stemsThrough",     _stemsThrough);
             xml.tag("upsideDown",       _upsideDown);
+            xml.tag("showTabFingering", _showTabFingering);
```

Comment:
See #3039 for fixing this in master, now that tst_guitarpro is enabled there again